configure.ac

```diff
@@ -277,13 +277,13 @@ AC_MSG_RESULT($asan_status)
 # User and group for running cockpit web server (cockpit-tls or -ws in customized setups)
 
 AC_ARG_WITH(cockpit_user,
-	    AS_HELP_STRING([--with-cockpit-user=<user>],
-			   [User for running cockpit (root)]
+           AS_HELP_STRING([--with-cockpit-user=<user>],
```

Comment:
Same here -- consistent indentation is good, but should be a separate commit

Reply:
this was an accident, i think

Reply:
I know what happened, because it just happened again: I have a habit of using `git show` + copy + `patch -R` + `git commit --amend` to get rid of things that I don't want in commits, and sometimes it turns tabs into spaces.